client/src/main/java/io/pravega/client/ClientConfig.java

```diff
@@ -73,6 +75,9 @@ public ClientConfig build() {
                 controllerURI = URI.create("tcp://localhost");
             }
             extractCredentials();
+            if (credentials == null) {
+                log.info("The credentials are not specified/could not be extracted");
```

Comment:
1. "specified or could not"
2. Add a period at the end of the message.

Reply:
Fixed.